target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-magic-2-wifi-next.dts

```diff
@@ -192,6 +149,14 @@
 	status = "okay";
 };
 
+&ethphy0 {
+	status = "disabled";
```

Comment:
why disabled?

Reply:
Because they were deleting the node which makes no sense since you can just disable it